docs/build/guides/dapps/soroban-contract-init-template.mdx

```diff
@@ -29,13 +29,15 @@ The official template maintained by Stellar Development Foundation (SDF), as use
 
 (You may wonder why it makes this unpopular choice. A fair question! The team wanted to balance actual utility with broad approachability. Not everyone learning Stellar and Soroban is familiar with React, or any other UI library. It also demonstrates that core Soroban libraries all work with any JavaScript project.)
 
-But note that the CLI's `contract init` command was made to work with _any_ frontend template. It need not even live on GitHub! Simply pass another valid git URL to the command, and it will clone that instead of the official repository:
+To use other templates, we will clone them from their repositories, and then copy these files into the root of the existing `soroban-hello-world` directory:
 
 ```bash
 # the following is NOT a valid git repository, and will not work 😄
-stellar contract init my-new-project --frontend-template https://whatever.cool/project/you/want
+git clone https://whatever.cool/project/you/want
```

Comment:
It's pretty important not to link out to websites or repos that don't exist or we don't have control or some trust in. Anyone owning this domain could host a malicious project at this path in the hope of capturing folks who happen to copy and paste this code snippet.

The only domain we should link to as an example should be example.org which is maintained by IANA for that purpose.

Reply:
Let's use the real repo. I added in the latest commit.